### PR TITLE
Update docs regarding coral enabled default

### PR DESCRIPTION
**Pull request by programmiri**
# Changes

Since release 2.4.0 the coral preview is enabled by default. This PR updates:

- the docs how to setup Klaw from source -> informs user that React UI is enabled by default and how to disabled it if they want to
- the release notes for 2.4.0 -> same as above it informs user how to disable the preview if they want to

### CODE REVIEW

File: docs/docs/howto/installation/run-source.rst [hunk 1]
```diff
@@ -10,7 +10,10 @@ Klaw is a Java project. Follow the below steps to run this project from the sour
 2. Experimental UI Configuration
    A redesigned user interface for Browse topics, providing a new look and feel for managing Kafka topics. It consists of changes to the layout, styling, and overall appearance of the interface, as well as improvements to make browsing and working topics easier, more efficient and accessible. This new look and feel is achieved by switching to React.
 
-    - Configure the property ``klaw.coral.enabled`` in the ``application.properties`` file to `true` in the module: core.
+   The redesigned UI is enabled by default since Release 2.4.0. If you want to use Klaw without the preview for now,
+   you can disabled it:
```
Comment (harshini-rangaswamy) [inline docs/docs/howto/installation/run-source.rst:14]:
```suggestion
   you can disable it:
```

File: docs/releases/release240.rst [hunk 2]
```diff
@@ -34,10 +34,12 @@ In this release, we have redesigned some key user interfaces using React to enha
   * *Connector restart capability*: Introducing the ability to restart Kafka connectors and their associated tasks, enabling efficient management and troubleshooting.
 
 
-To preview the new Klaw user interface, open the ``application.properties`` file on the Klaw **core** module, and set the value of the following property to ``true`` (Effective this version, it is true by default):
+To disabled the preview for the new Klaw user interface, open the ``application.properties`` file on the Klaw
```
Comment (harshini-rangaswamy) [inline docs/releases/release240.rst:37]:
```suggestion
To disable the preview for the new Klaw user interface, open the ``application.properties`` file on the Klaw
```

**Review by harshini-rangaswamy — CHANGES_REQUESTED** [review]
@programmiri, just a minor change.

**Comment by programmiri** [comment]
Thank you @harshini-rangaswamy! Not sure what was wrong with me and the word "disable" yesterday 😅  Can you check again and approve if everything is ok? 🙏

**Review by harshini-rangaswamy — APPROVED** [review]
LGTM!

**Pull request merged**